src/services/config-loader.ts

```diff
@@ -82,8 +84,8 @@ export class MainConfigLoaderService {
     let counter = 0;
 
     for (const service of this.services) {
-      let name = "";
-      let email = "";
+      let name = '';
+      let email = '';
```

Comment:
Do not change this please.
And also merge with main branch to avoid too many changes in this file.